src/components/RoomNameInput/index.js

```diff
@@ -0,0 +1,125 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import CONST from '../../CONST';
+import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import TextInput from '../TextInput';
+
+const propTypes = {
+    /** Callback to execute when the text input is modified correctly */
+    onChangeText: PropTypes.func,
+
+    /** Room name to show in input field. This should include the '#' already prefixed to the name */
+    value: PropTypes.string,
+
+    /** Whether we should show the input as disabled */
+    disabled: PropTypes.bool,
+
+    /** Error text to show */
+    errorText: PropTypes.string,
+
+    ...withLocalizePropTypes,
+
+    /** A ref forwarded to the TextInput */
+    forwardedRef: PropTypes.func,
+};
+
+const defaultProps = {
+    onChangeText: () => {},
+    value: '',
+    disabled: false,
+    errorText: '',
+    forwardedRef: () => {},
+};
+
+class RoomNameInput extends Component {
+    constructor(props) {
+        super(props);
+
+        this.setModifiedRoomName = this.setModifiedRoomName.bind(this);
+        this.setSelection = this.setSelection.bind(this);
+
+        this.state = {
+            selection: {start: 0, end: 0},
```

Comment:
In mWeb Safari with the `selection` value set to `{start: 0, end: 0}`, the input is autofocus on the room settings page.

While the expected is the input is not autofocused on mount the room settings page, and because the input has value, just set this to `undefined`. So the input is not autofocus and the cursor is not set in the first line.